lib/simulator/Simulator.cpp

```diff
@@ -1,158 +1,36 @@
-  UseRealMillis();
-
-  // Note: use this line if you need to change the rendering config
-  // OgreBites::getNativeConfigDialog()->display();
-  OgreBites::ApplicationContext::setup();
-
-  // register for input events
-  addInputListener(this);
-
-  // get a pointer to the already created root
-  Ogre::Root *root = getRoot();
-  Ogre::SceneManager *scnMgr = root->createSceneManager();
-  // register our scene with the RTSS
-  Ogre::RTShader::ShaderGenerator *shadergen =
-      Ogre::RTShader::ShaderGenerator::getSingletonPtr();
-  shadergen->addSceneManager(scnMgr);
-
-  Ogre::SceneNode *camNode = scnMgr->getRootSceneNode()->createChildSceneNode();
-  camNode->setPosition(0, SimulatorLightController::inchesToCoords(69),
-                       SimulatorLightController::feetToCoords(15));
-  camNode->lookAt(
-      Ogre::Vector3(0, SimulatorLightController::feetToCoords(0), 0),
-      Ogre::Node::TransformSpace::TS_WORLD);
-
-  Ogre::Camera *cam = scnMgr->createCamera("cam1");
-  cam->setNearClipDistance(5);
-  camNode->attachObject(cam);
-
-  Ogre::Viewport *vp = getRenderWindow()->addViewport(cam);
-  vp->setBackgroundColour(Ogre::ColourValue(0, 0, 0));
-  cam->setAspectRatio(Ogre::Real(vp->getActualWidth()) /
-                      Ogre::Real(vp->getActualHeight()));
-
-  Ogre::Plane wall(Ogre::Vector3::UNIT_Y, 0);
-  Ogre::MeshManager::getSingleton().createPlane(
-      "wall", Ogre::ResourceGroupManager::DEFAULT_RESOURCE_GROUP_NAME, wall,
-      /* width */ SimulatorLightController::feetToCoords(30),
-      /* height */ SimulatorLightController::feetToCoords(30), 20, 20, true, 1,
-      5, 5, Ogre::Vector3::UNIT_Z);
-  Ogre::Entity *wallEntity = scnMgr->createEntity("wall");
-  wallEntity->setCastShadows(false);
-  wallEntity->setMaterialName("Template/White");
-  Ogre::SceneNode *wallNode =
-      scnMgr->getRootSceneNode()->createChildSceneNode();
-  wallNode->setPosition(0, 0, 0);
-  wallNode->attachObject(wallEntity);
-
-  for (int i = 0; i < 6; i++) {
-    Ogre::Entity *pole_entity = scnMgr->createEntity("column.mesh");
-    // With very many lights, turning on shadows for the poles makes this too
-    // slow.
-    pole_entity->setCastShadows(true);
-    pole_entity->setMaterialName("Template/White");
-    Ogre::AxisAlignedBox bounding_box = pole_entity->getBoundingBox();
-    Ogre::Vector3 size = bounding_box.getSize();
-
-    Ogre::SceneNode *pole_node =
-        scnMgr->getRootSceneNode()->createChildSceneNode();
-    pole_node->setPosition(
-        SimulatorLightController::feetToCoords(7.5) * cos(3.14 / 3.0 * i), 0,
-        SimulatorLightController::feetToCoords(7.5) * sin(3.14 / 3.0 * i));
-    pole_node->attachObject(pole_entity);
-    pole_node->setScale(
-        Ogre::Vector3(SimulatorLightController::inchesToCoords(2) / size.x,
-                      SimulatorLightController::feetToCoords(10) / size.y,
-                      SimulatorLightController::inchesToCoords(2) / size.z));
+int main(int argc, char** argv) {
```

Comment:
You can declare main as `int main()`, no need to have the unused `argc` and `argv`

Reply:
Done.